src/chrome/content/rules/Adlibris.xml

```diff
@@ -1,19 +1,12 @@
+<ruleset name="Adlibris">
+	<target host="adlibris.com" />
+	<target host="www.adlibris.com" />
 
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://www.capris.no/ => https://www.capris.no/: (51, "SSL: no alternative certificate subject name matches target host name 'www.capris.no'")
-Fetch error: http://capris.no/ => https://www.capris.no/: (51, "SSL: no alternative certificate subject name matches target host name 'www.capris.no'")
+	<target host="capris.no" />
+	<target host="www.capris.no" />
 
-Disabled by https-everywhere-checker because:
-Fetch error: http://www.capris.no/ => https://www.capris.no/: (60, 'SSL certificate problem: certificate has expired')
-Fetch error: http://capris.no/ => https://www.capris.no/: (60, 'SSL certificate problem: certificate has expired')
--->
-<ruleset name="Adlibris/Capris" platform="mixedcontent" default_off='failed ruleset test'>
-  <target host="www.adlibris.com" />
-  <target host="adlibris.com" />
-  <target host="www.capris.no" />
-  <target host="capris.no" />
+	<securecookie host=".+" name=".+" />
 
-  <rule from="^http://(?:www\.)?(adlibris\.com|capris\.no)/"
-          to="https://www.$1/"/>
+	<rule from="^http://(www\.)?capris\.no/" to="https://www.adlibris.com/" />
```

Comment:
Please add a comment for it.

Reply:
This doesn't seem to be required anymore.

Reply:
https://travis-ci.org/EFForg/https-everywhere/jobs/497516138#L569-L570

Reply:
For some inexplicable reason I can access both without problem.

Reply:
Done.